man/systemd.special.xml

```diff
@@ -488,6 +489,17 @@
             this target unit, for compatibility with SysV.</para>
           </listitem>
         </varlistentry>
+        <varlistentry>
+          <term><filename>rdt.target</filename></term>
+          <listitem>
+            <para>A special target unit for starting rdtd service.</para>
+
+            <para>Applications congfigured to be a member of some RDT group should
+            gain an <varname>After=</varname> and <varname>Wants=</varname> dependencies
```

Comment:
s/should get/will get/

Please mention `<varname>RDTControlGroup=</varname>` setting here explicitly.